ENIGMAsystem/SHELL/Universal_System/image_formats.cpp

```diff
@@ -320,7 +320,7 @@ int image_save_png(string filename, const unsigned char* data, unsigned width, u
         file.close();
     }
     free(buffer);
-	free(bitmap);
+    delete[] bitmap;
```

Comment:
Good job, `new` should always go with `delete` and `free()` only goes with `malloc()`. This file does not have any malloc calls, so there should be no calls to free:
https://stackoverflow.com/a/4061522